common/messages.py

```diff
@@ -355,3 +391,13 @@ def parse_message(message_bytes):
         raise RuntimeError('Invalid message type received: id 0x%04X was not found in _message_map' % msg_id)
     msg = _message_map[msg_id].from_bytes(message_bytes)
     return msg
+
+
+def parse_message_string(message_str):
```

Comment:
Better name: parse_message_from_string. Also rename parse_message to parse_message_from_bytes